addons/account/views/partner_view.xml

```diff
@@ -113,7 +113,7 @@
                         </group>
                         </page>
                     </notebook>
-                    <field name="note" placeholder="Legal Notes..."/>
+                    <field name="note" class="oe-bordered-editor" placeholder="Legal Notes..."/>
```

Comment:
> This change is only be a visual enhancement
and should not prevent the function of said html fields
event if the views are not updated.

I stumbled on the word "event" because of the typo.